src/Contrib/Authentication/OpenIdConnect/Masa.Contrib.Authentication.OpenIdConnect.EFCore/Repositories/ApiScopeRepository.cs

```diff
@@ -39,6 +39,7 @@ public async Task<PaginatedList<ApiScope>> GetPaginatedListAsync(int page, int p
                          .Include(apiScope => apiScope.UserClaims)
                          .ThenInclude(apiScope => apiScope.UserClaim)
                          .Include(apiScope => apiScope.Properties)
+                         .AsSplitQuery()
```

Comment:
I don't think it needs split query, you just need control left join and inner join.